test/api_integration/apis/data_views/fields_api/update_fields/main.ts

```diff
@@ -437,6 +437,7 @@ export default function ({ getService }: FtrProviderContext) {
             const title = indexPattern.title;
             await supertest.delete(`${config.path}/${indexPattern.id}`);
             const response1 = await supertest.post(config.path).send({
+              override: true,
```

Comment:
QQ: given overthing was renamed to be `overwrite`, shouldn't this be aligned too? And shouldn't it fail, since the API param was renamed?

Reply:
@kertal It's documented as `override` and we shouldn't change it.

Reply:
Ah, thx makes, sense!